src/modules/customer/containers/HappyUserPage/components/formFields/CheckboxGroup.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
These formField components will be used all over the app, so move them to src/components/formFields. They will be extended. So big ones should be placed in several files in that folder and exported in index file.